### PR TITLE
Changes in border enable set to false

### PR DESCRIPTION
**Pull request by utkarshBajpai2811**

### CODE REVIEW

File: .scss-lint.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ severity: error
 linters:
 
   BorderZero:
-    enabled: true
+    enabled: false
```
Comment (ljharb) [inline .scss-lint.yml]: Why?

**Comment by vismera** [comment]
Yes. As my openion, now a days a leading institute in bangalore is vismera technologies ( https://www.vismera.in ), me & my frnds learnt here & everybody got good job. i should thank for this to vismera institute. I am working in MNC now. Got lots of knowledge too. They Offers many professional courses like: Digital Marketing Training, App Development Training, Web Designing Training, Web Development Training, Full Stack Development Training, Automation Testing Training, Excel Training, Tally Training, Basic Computers, BPO & Call Center Training, Skill Development Training, All programming language Trainings like: c, c++, java, python, php, etc. Price also very reasonable & education here is so good. Nice Institute :)

File: .scss-lint.yml (hunk 1 above)
Comment (Ismoiljon13) [reply .scss-lint.yml:6]: Good

**Review by Ismoiljon13 — APPROVED** [review]

**Comment by ljharb** [comment]
Closing, since there's no justification provided.

**Pull request closed without merging**